docs/sources/operators-guide/monitor-grafana-mimir/collecting-metrics-and-logs.md

```diff
@@ -107,35 +107,27 @@ metaMonitoring:
             app.kubernetes.io/name: kube-state-metrics
 ```
 
-#### Sending metrics back into Mimir/GEM
+#### Send metrics back into Mimir/GEM
 
 You can also send the collected metamonitoring metrics to the installation of Mimir or GEM.
-The configuration varies slightly for GEM and Mimir.
 
-If you have deployed Mimir, then the Helm chart values should look like the following example.
-Replace `HELM_RELEASE_NAME` with the name of your Helm release:
+When you leave the `metamonitoring.grafanaAgent.metrics.remote.url` field empty,
+then the chart automatically fills in the address of the GEM gateway Service
+or the Mimir NGINX Service.
 
-```yaml
-metaMonitoring:
-  serviceMonitor:
-    enabled: true
-  grafanaAgent:
-    enabled: true
-    installOperator: true
-
-  metrics:
-    remote:
-      url: "http://<HELM_RELEASE_NAME>-mimir-nginx.mimir.svc/api/v1/push"
-      headers:
-        X-Scope-OrgID: metamonitoring
-```
+If you have deployed Mimir, and `metamonitoring.grafanaAgent.metrics.remote.url` is not set,
+then the metamonitoring metrics are be sent to the Mimir cluster.
+You can query these metrics using the HTTP header X-Scope-OrgID: metamonitoring
 
-If you have deployed GEM, then the URL will point to the GEM gateway instead of the nginx Deployment.
-If you are using the GEM authentication model, then you also need to provide a Secret with the
-authentication token for a tenant. Refer to [Credentials](#credentials) for setting up the Secret.
+If you have deployed GEM, then there are two cases:
```

Comment:
```suggestion
If you have deployed GEM, then there are two contexts:
```

Reply:
I am not sure cases or contexts is the best option wrt veriage. Still thinking...

Reply:
"... two alternatives:" ?

Reply:
alternatives sounds good